.travis.yml

```diff
@@ -3,6 +3,6 @@ before_install:
   - sed -i.bak -e 's|https://nexus.codehaus.org/snapshots/|https://oss.sonatype.org/content/repositories/codehaus-snapshots/|g' ~/.m2/settings.xml
 script:
         mvn clean test &&
-        mvn clean test -Pall
+        mvn clean test -Pall -DfailIfNoTests=false
```

Comment:
Well it hit me hard when I just run `mvnci -Pall`. Used to be working before. As I use I just want to run build the simplest way I can. I find the profile, I enable it, but I shouldn't be bothered by other failures. 

We can keep empty `all` profile in the parent `pom.xml` for documentation purposes (maybe even with some explanation what it is used for)